lib/simple_form.rb

```diff
@@ -265,6 +265,49 @@ def self.setup
     @@configured = true
     yield self
   end
+
+  # Includes a component to be used by Simple Form. Methods defined in a
+  # component will be exposed to be used in the wrapper as Simple::Components
+  #
+  # Examples
+  #
+  #    # The application needs to tell where the components will be.
+  #    Dir[Rails.root.join('config/initializers/simple_form_components/**/*.rb')].each { |f| require f }
+  #
+  #    # Create a custom component in the path specified above.
+  #    # config/initializers/simple_form_components/input_group.rb
```

Comment:
to follow rails naming convention the file should be prefix with `_component.rb` => `input_group_component.rb`